plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/part/AttributePart.java

```diff
@@ -117,6 +124,16 @@ public void setSelected(int value) {
             }
         }
         columnLabel.repaint();
+
+        if (value != EditPart.SELECTED_NONE) {
+            if (this.getViewer() instanceof ERDGraphicalViewer && associatedRelationsHighlighing == null) {
```

Comment:
These `if` statements can be merged.

Reply:
This two if statement's can't be merged, it will break the logic.